examples/functions-with-defaultable-parameters/functions_with_defaultable_parameters.bal

```diff
@@ -1,13 +1,13 @@
 import ballerina/io;
 
-// This function takes salary, annual increment, and bonus rate as input 
+// This function takes base salary, annual increment and bonus rate as input
 // parameters and prints them. The first parameter `baseSalary` is a required
 // parameter. The next two parameters `annualIncrement` and `bonusRate` are 
-// two defaultable parameters. They are optional parameters with a default 
-// value of 20 and 0.02 respectively. Defaultable parameters must always be
+// two defaultable parameters. They are optional parameters which have default
```

Comment:
```suggestion
// two defaultable parameters. They are optional parameters that have default
```

You would normally use "that" in this context. The word "which" is used when you are adding additional information to the sentence that is not necessary for the sentence to be complete. On the other hand, "that" is used to add necessary content for sentence completion. Also, when you use "which" in this context, you would normally have a comma in front of it.